File: tecjobs-api/src/main/java/com/raffdevs/tecjobsapi/dtos/CreateCompanyDTO.java
package com.raffdevs.tecjobsapi.dtos;

import java.io.Serializable;

public class CreateCompanyDTO implements Serializable {
    final private String name;
    final private String aboutUs;

    public CreateCompanyDTO(String name, String aboutUs) {
        this.name = name;
        this.aboutUs = aboutUs;
    }

    public String getName() {
        return name;
    }

    public String getAboutUs() {
        return aboutUs;
    }
}

File: tecjobs-api/TecJobsAPI/DTO/CreateCompanyDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TecJobsAPI.DTO
{
    public record CreateCompanyDTO
    {
        public string Name { get; init; }
        public string AboutUs { get; init; }

        public CreateCompanyDTO(string name, string aboutUs)
        {
            Name = name;
            AboutUs = aboutUs;
        }
    }
}